.github/workflows/enforce_conventional_commit.yml

```
@@ -0,0 +1,14 @@
+name: Validate
+on: [pull_request]
+
+jobs:
+  validate_comments:
+    name: 'Validate Conventional Commits'
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v1
```

```suggestion
      - uses: wagoid/commitlint-github-action@v2
```

Latest is version 2: https://github.com/wagoid/commitlint-github-action